docs/verify/local-testing-mode.ipynb

```diff
@@ -76,14 +76,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "id": "c5edb89a-65e6-40fa-a541-bbdcf6cc7043",
    "metadata": {
     "tags": [
      "ignore-warnings"
     ]
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/frankharkins/Documents/documentation/.tox/py311/lib/python3.11/site-packages/qiskit_ibm_runtime/session.py:156: UserWarning: Session is not supported in local testing mode or when using a simulator.\n",
```

Comment:
```suggestion
      "python3.11/site-packages/qiskit_ibm_runtime/session.py:156: UserWarning: Session is not supported in local testing mode or when using a simulator.\n",
```

This notebook was not run before so had no outputs. I think it's helpful to show this warning so users understand it's expected, but I've edited out my device-specific information.

Reply:
Sure!